controls/cis_rhel10.yml

```diff
@@ -449,13 +447,6 @@ controls:
       - file_groupowner_user_cfg
       - file_owner_user_cfg
       - file_permissions_user_cfg
-    related_rules:
```

Comment:
Related rules don't are not selected, so there is no need to keep them in the newly added default.profile for RHEL10